modules/rational-defaults.el

```diff
@@ -13,7 +13,7 @@
 (fset 'yes-or-no-p 'y-or-n-p)
 
 ;; Turn on recentf mode
-(recentf-mode 1)
+(add-hook 'after-init-hook (lambda () (recentf-mode 1)))
```

Comment:
I think you *should* be able to get away with this simplification:
```suggestion
(add-hook 'after-init-hook #'recentf-mode)
```

Reply:
Just tested and it works well 👍🏻 thanks for the suggestion!